samples/python/beam_search_causal_lm/beam_search_causal_lm.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import openvino_genai
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('model_dir')
+    parser.add_argument('prompts', nargs='+')
+    args = parser.parse_args()
+
+    device = 'CPU'  # GPU can be used as well
+    pipe = openvino_genai.LLMPipeline(args.model_dir, device)
+
+    config = openvino_genai.GenerationConfig()
+    config.max_new_tokens = 20
+    config.num_beam_groups = 3
+    config.num_beams = 15
+    config.num_return_sequences = config.num_beams * len(args.prompts)
```

Comment:
```suggestion
    config.num_return_sequences = config.num_beams
```
I suppose default for `num_return_sequences` should be `config.num_beams`, if other value is not explicitly specified. So, can we drop setting this variable at all?

Reply:
The default is 1

Reply:
for HF or OV?

Reply:
Both of them

Reply:
ok, but at least we need to apply current suggestion?

Reply:
That would discard other prompts results making it equivalent to accepting a single prompt only as the input.